fineract-provider/build.gradle

```diff
@@ -274,10 +252,50 @@ bootJar {
     classifier = ''
 }
 
-bootWar {
-    enabled = false
+jib {
+    from {
+        // image = 'azul/zulu-openjdk-alpine:17-jre' // once we get rid of OpenJPA we could reduce space significantly
+        image = 'azul/zulu-openjdk-alpine:17'
+    }
+
+    to {
+        image = 'fineract'
+        tags = [
+            "${releaseVersion}",
+            'latest'
+        ]
+    }
+
+    container {
+        creationTime = 'USE_CURRENT_TIMESTAMP'
+        jvmFlags = [
+            '-Duser.home=/data',
+            '-Xmx1G',
+            '-Xms1G',
+            '-XshowSettings:vm',
+            '-XX:+UseContainerSupport',
+            '-XX:+UseStringDeduplication',
+            '-XX:MinRAMPercentage=25',
+            '-XX:MaxRAMPercentage=80'
+        ]
+        args = [
+            '-Dspring.profiles.active=basicauth',
```

Comment:
Could we make this env parameter for the image, but default to basicauth? That way we could use the same image for all the different profiles (twofactor, oauth etc) and also run tests for each of them?